include/pcmki/pcmki_status.h

```diff
@@ -50,7 +50,7 @@ int pcmk__status(pcmk__output_t *out, cib_t *cib,
                  enum pcmk__fence_history fence_history, uint32_t show,
                  uint32_t show_opts, const char *only_node,
                  const char *only_rsc, const char *neg_location_prefix,
-                 bool simple_output);
+                 bool simple_output, int timeout_ms);
```

Comment:
guint makes more sense

Reply:
Yeah you're right. Normally I lean slightly toward "don't use types to mask programmer error", but it's passed as the `message_timeout_ms` (`guint`) argument of `pcmk__pacemakerd_status()`.